swap/swap.go

```diff
@@ -33,17 +33,21 @@ import (
 // A node maintains an individual balance with every peer
 // Only messages which have a price will be accounted for
 type Swap struct {
-	stateStore state.Store        //stateStore is needed in order to keep balances across sessions
-	lock       sync.RWMutex       //lock the balances
-	balances   map[enode.ID]int64 //map of balances for each peer
-	service    *SwapService
+	stateStore          state.Store        // stateStore is needed in order to keep balances across sessions
+	lock                sync.RWMutex       // lock the balances
+	balances            map[enode.ID]int64 // map of balances for each peer
+	service             *SwapService
+	paymentThreshold    int64 // balance difference required for requesting cheque
+	disconnectThreshold int64 // balance difference required for dropping peer
 }
 
 // New - swap constructor
 func New(stateStore state.Store) (swap *Swap) {
 	swap = &Swap{
-		stateStore: stateStore,
-		balances:   make(map[enode.ID]int64),
+		stateStore:          stateStore,
+		balances:            make(map[enode.ID]int64),
+		paymentThreshold:    DefaultPaymentThreshold,
```

Comment:
The idea is good but I wonder, does it make sense to have more than one threshold in parallel? I personally believe this will not happen, therefore we don't need a `Default...Threshold` - we just need one, and then we don't need to add it to the `Swap` object and can always refer directly to the constant.

What do you think? Can you think of a situation where it makes sense to have `Default...`?

Reply:
I suppose based on configuration or initialization parameters, it might make sense for a node to specify its own thresholds.

Should this specification be optional, it would make sense to have default values.

Of course, the code must be adapted in order to have this option.

Reply:
ok